lib/client.js

```diff
@@ -76,4 +77,39 @@ Client.prototype.listArchives = function() {
 Client.prototype.deleteArchive = function() {
 };
 
+Client.prototype.dial = function (opts, cb) {
+  request.post({
+    // TODO: only works while apiUrl is always up to the domain, without the ending slash
+    url: this.c.apiUrl + '/v2/project/' + this.c.apiKey + this.c.endpoints.dial,
```

Comment:
i understand that the URL has the apiKey in it, so you can't just store the literal value in L13, but you actually have a chance to modify this during the call to `configure()`. during that call, the default properties for the config are merged with any specific values on the instance. therefore you could use that to read the default endpoint and substitute the specific apiKey into the URL.